lib/dal/src/attribute/value.rs

```diff
@@ -549,93 +550,121 @@ impl AttributeValue {
         // progress). To handle this here, we grab a read lock, which will be
         // locked for writing in the dependent values update job while the
         // execution result is being written to the graph.
+
+        // BRIT HERE
```

Comment:
HELLO BRIT (probably should remove this)

Reply:
ah yes thank you! 🤦‍♀️

Reply:
Nah let's keep it 😤